mmeval/metrics/char_recall_precision.py

```diff
@@ -0,0 +1,103 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import re
+from difflib import SequenceMatcher
+from typing import Dict, Sequence, Tuple
+
+from mmeval.core import BaseMetric
+
+
+class CharRecallPrecision(BaseMetric):
+    """Calculate the char level recall & precision.
+
+    Args:
+        letter_case (str): There are three options to alter the letter cases
+            - unchanged: Do not change prediction texts and labels.
+            - upper: Convert prediction texts and labels into uppercase
+                     characters.
+            - lower: Convert prediction texts and labels into lowercase
+                     characters.
+            Usually, it only works for English characters. Defaults to
+            'unchanged'.
+        valid_symbol (str): A regular expression to filter out invalid or
+            not cared characters. Defaults to '[^A-Z^a-z^0-9^\u4e00-\u9fa5]'.
```

Comment:
`\u4e00-\u9fa5` renders to strange symbols on api docs. Shourd we add a 'r' at the beginging of this docstring?